### ISSUE TITLE
Question about traces

### ISSUE THREAD

**Issue by lferna** [quarkiverse/quarkiverse-logging-json#36]
Hi,

First of all, thank you for your effort to have this library in quarkus.

I want to know if there is a way to customize traces. These are the traces if we set the quarkus tracing library:

`{"timestamp":"2020-10......,"mdc":{"spanId":"521581a8576ba5b3","traceId":"521581a8576ba5b3","sampled":"false"}}`

And I want:

`{"timestamp":"2020-10-16T18:16:47.928+02:00", "correlatorId":"521581a8576ba5b3","transactionId":"521581a8576ba5b3"....}`

So I want to remove "mdc" object in logs and then, rename the key names. 

Is there any way to do that? 

Regards.

**Comment by SlyngDK** [comment]
Hi @lferna 

You will be able to disable logging of the mdc using application.config
`quarkus.log.console.json.fields.mdc.enabled=false`

I have not yet implemented support for providing a custom `io.quarkiverse.loggingjson.JsonProvider`, that is the next feature I will implement. 
It will getting to look something like this:
```java
@Provider
@ApplicationScoped
public class CustomTraceJsonProvider implements io.quarkiverse.loggingjson.JsonProvider {
    @Override
    public void writeTo(JsonGenerator generator, ExtLogRecord event) throws IOException {
       String traceId = event.getMdc("traceId");
       if(traceId != null) {
           generator.writeStringField("correlatorId", traceId);
       }
    }
}
```
Is that what you are looking for?

**Comment by lferna** [comment]
Hi @SlyngDK 
Thank you for your response. Yes, that is what I'm looking for!

Any ideas until you have that solution implemented? Is there any possibility on my side to override one of the providers with that code?

Regards,

**Comment by SlyngDK** [comment]
This is now released in 0.1.6.
